### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by WaydeTheunissen**

### CODE REVIEW

File: WITW.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+    <div class="container theme-showcase main_container" role="main">
+    	<section class="focus_block">
+    		<img src="images/Alpha_Geo-Location_Feature_Graphic.png" alt="" />
+    	</section>
+      	<!-- Main jumbotron for a primary marketing message or call to action -->
+        <div class="jumbotron">
+            <p>Where in the World is a Google Maps tutorial implemented in a simple Android application. Eventually we're going to fork this application and create a more useful location awareness application.</p>\
+            <div class="image_container">
+                <img src="images/Alpha_Map_Screenshot_Phone.jpg" alt="" />
```
Comment (nicsaal) [inline WITW.html]: I'd prefer this image removed from page. It divides the text into 2 paragraphs and takes up a lot of screen real estate if viewing from the mobile app.

File: WITW.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            <li><a href="#contact">Contact</a></li>
+          </ul>
+        </div>
+        <!--/.nav-collapse -->
+      </div>
+    </nav>
+
+    <div class="container theme-showcase main_container" role="main">
+    	<section class="focus_block">
+    		<img src="images/Alpha_Geo-Location_Feature_Graphic.png" alt="" />
```
Comment (nicsaal) [inline WITW.html]: Two things:
1. Can you change this to point to feature_graphic.png which I sent via email.
2. Can you upload feature_graphic.png to images folder.

**Comment by oliversaal** [comment]
I'm not sure why there is another PR - couldn't the changes have been applied to the first PR. I also don't see any description - were all the changes in the document covered; was their any places it wasn't possible etc.

It's very difficult to see the diff or alterations in this form. Can you merge it into the first PR?

File: WITW.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    	</section>
+      	<!-- Main jumbotron for a primary marketing message or call to action -->
+        <div class="jumbotron">
+            <p>Where in the World is a Google Maps tutorial implemented in a simple Android application. Eventually we're going to fork this application and create a more useful location awareness application.</p>\
```
Comment (oliversaal) [inline WITW.html]: Nice catch Nic!

**Pull request closed without merging**